modules/cert-manager-install-console.adoc

```diff
@@ -0,0 +1,44 @@
+// Module included in the following assemblies:
+//
+// * security/cert_manager_operator/cert-manager-operator-install.adoc
+
+:_content-type: PROCEDURE
+[id="cert-manager-install-console_{context}"]
+= Installing the {cert-manager-operator} using the web console
+
+You can use the web console to install the {cert-manager-operator}.
+
+.Prerequisites
+
+* You have access to the cluster with `cluster-admin` privileges.
+* You have access to the {product-title} web console.
+
+.Procedure
+
+. Log in to the {product-title} web console.
+
+. Navigate to *Operators* -> *OperatorHub*.
```

Comment:
@bergerhoffer  For navigation through menus, the character should be a **>** rather than a **->**.